### PR TITLE
Added Essentiel B TV Remote

### PR DESCRIPTION
**Pull request by atao**
Essentiel B is Boulanger's distributor brand

### CODE REVIEW

File: TVs/EssentielB/EssentielB.ir [hunk 1]
```diff
@@ -0,0 +1,71 @@
+Filetype: IR signals file
+Version: 1
+#
+# TV LED ESSENTIELB Android TV
+# Essentiel B is Boulanger's distributor brand
+#
+name: On_off
```
Comment (darmiel) [inline TVs/EssentielB/EssentielB.ir:7]:
```suggestion
name: POWER
```

File: TVs/EssentielB/EssentielB.ir [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+address: 0F 00 00 00
+command: 54 00 00 00
+# 
+name: Mute
+type: parsed
+protocol: RCA
+address: 0F 00 00 00
+command: FC 00 00 00
+# 
+name: Pr_up
```
Comment (darmiel) [inline TVs/EssentielB/EssentielB.ir:19]:
```suggestion
name: CH+
```

File: TVs/EssentielB/EssentielB.ir [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+type: parsed
+protocol: RCA
+address: 0F 00 00 00
+command: B4 00 00 00
+# 
+name: Pr_down
```
Comment (darmiel) [inline TVs/EssentielB/EssentielB.ir:25]:
```suggestion
name: CH-
```

**Review by darmiel — CHANGES_REQUESTED** [review]
Looks good so far, but can you please change some names in the TV remote to match the [naming scheme](https://github.com/logickworkshop/Flipper-IRDB#organization)?

File: TVs/EssentielB/EssentielB.ir (hunk 3 above)
Comment (atao) [reply TVs/EssentielB/EssentielB.ir:25]: I will create a new pull request with fix.

**Comment by darmiel** [comment]
Looks good, thank you! :)

**Pull request merged**